src/main/java/com/example/bookstorewebapp/service/impl/BookServiceImpl.java

```diff
@@ -37,4 +37,15 @@ public BookDto findById(Long id) {
         );
         return bookMapper.toDto(book);
     }
+
+    @Override
+    public void deleteById(Long id) {
+        bookRepository.deleteById(id);
+    }
+
+    public BookDto updateById(Long id, CreateBookRequestDto requestDto) {
+        Book book = bookMapper.toModel(requestDto);
+        book.setId(id);
+        return bookMapper.toDto(bookRepository.save(book));
+    }
```

Comment:
I think the main idea of "update" is to modify an existing entity. It may be better to include a check to see if there is a book with the ID that the user is attempting to update.